tests/test_cli_clean.py

```diff
@@ -206,7 +206,7 @@ def test_clean_abort(notebooks_dir):
 
     result = CliRunner().invoke(app, ["clean", str(notebooks_dir)], input="n")
     assert result.exit_code == 1
-    assert result.stdout.strip().endswith("Aborted.")
+    assert "Aborted" in result.stdout.strip()[-8:]
```

Comment:
this magic number feels a little finicky. why not just `"Aborted" in result.stdout`?

Reply:
Maybe I'm just paranoid, but I want to make this relatively strict to avoid false positives from "Aborted" showing up some other way earlier in stdout.